modules/site-health/audit-autoloaded-options/load.php

```diff
@@ -117,3 +117,44 @@ function perflab_aao_autoloaded_options_size() {
 	global $wpdb;
 	return (int) $wpdb->get_var( 'SELECT SUM(LENGTH(option_value)) FROM ' . $wpdb->prefix . 'options WHERE autoload = \'yes\'' );
 }
+
+/**
+ * Fetch autoload top list.
+ *
+ * @since n.e.x.t
+ *
+ * @return array autoloaded data - autoload size, entries, option and its names.
```

Comment:
Descriptions should start with a capital letter.
```suggestion
 * @return array Autoloaded data as option names and their sizes.
```